Library/SharedFunctions.swift

```diff
@@ -225,15 +225,9 @@ internal func sanitizedPledgeParameters(
   pledgeAmount: Double,
   shippingRule: ShippingRule?
 ) -> SanitizedPledgeParams {
-  var pledgeTotal = pledgeAmount
-  var shippingLocationId: String?
+  let shippingLocationId = (shippingRule?.location.id).flatMap(String.init)
 
-  if let shippingRule = shippingRule {
-    pledgeTotal = pledgeAmount.addingCurrency(shippingRule.cost)
```

Comment:
Previously we added the shipping cost to the total in this function, this has been extracted to happen outside of this function to make it easier to deal with the same total everywhere.